### ISSUE TITLE
Hyper should open at the cwd

### ISSUE THREAD

**Issue by Tyriar** [zeit/hyper#1213]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Ubuntu 16.10
- **Hyper.app version**: 1.0.0

## Issue

A VS Code user is attempting to configure hyper as an external terminal, we use the current working directory to solve this issue generally which doesn't appear to work for Hyper. I then checked `hyper --help` if there are any args that control this (which is less than ideal) but --help is not implemented (#1212).

**Comment by IanMitchell** [comment]
Does adding `hypercwd` to your plugin list fix this?

https://www.npmjs.com/package/hypercwd

**Comment by Tyriar** [comment]
@IanMitchell that's purely for launching a new tab within hyper itself isn't it? I need to launch a new instance externally at a particular directory, ideally relying on cwd and not some special tags so it doesn't need to be handled specially.

**Comment by ppot** [comment]
@Tyriar Yes it is. I'm currently working on allowing this possibility. At the same time of command invocation

**Comment by IanMitchell** [comment]
Ah, gotcha. My bad, didn't fully understand

**Comment by WillemLabu** [comment]
[@nsna's comment](https://github.com/zeit/hyper/issues/1165#issuecomment-267019160) on a different issue fixed this for me on Windows:

In a nutshell, `regedit`, browse to `HKEY_CLASSES_ROOT\Directory\Background\shell\Hyper\command`, and edit the value to:

```
C:\Users\{user}\AppData\Local\hyper\app-1.0.0\Hyper.exe "%V"
```

**Comment by Tyriar** [comment]
This should really work out of the box, at least for Unix-like systems where all terminals behave like this to my knowledge. Doing this may also fix the context menu issue #1165

**Comment by pchomik** [comment]
Is it possible to get this feature ?

**Comment by Stanzilla** [comment]
Do note that https://github.com/hharnisc/hypercwd is currently broken as of version 1.2.2, so if you are having issues with it and got here via a Google search, check their repo for an update.

**Comment by joglr** [comment]
Any update on this?

**Comment by octipus** [comment]
i`m interested in this as well

**Comment by 1silvertiger** [comment]
I would like this as well.

**Comment by FatehAK** [comment]
Any updates on this issue?

**Comment by AliNisarAhmed** [comment]
I installed hyper today and as an external terminal to vscode, it is opening in home directory, rather than the project directory :(

**Comment by linonetwo** [comment]
Use Github Desktop and press `ctrl + \`` will open hyper on that repo.

**Comment by rajatpatel2113** [comment]
I really need this. Why is this not supported yet? Even after two years? :(

**Comment by JoeEarly** [comment]
I'll add my vote to this

**Comment by Stanzilla** [comment]
Hyper is fairly unmaintained atm, so pull requests are very welcome!

**Comment by nikzasel** [comment]
Any update on this?

**Comment by Tsimopak** [comment]
Any update?

**Comment by danilodecanini** [comment]
any?

**Comment by Stanzilla** [comment]
my comment still stands, sorry.

**Comment by dexsper** [comment]
any?
